pkg/app/pipedv1/plugin/kubernetes/toolregistry/scripts.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toolregistry
+
+const kubectlInstallScript = `
+cd {{ .TmpDir }}
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v{{ .Version }}/bin/{{ .Os }}/{{ .Arch }}/kubectl
+mv kubectl {{ .OutPath }}
```

Comment:
These template variables come from the below comments.
https://github.com/pipe-cd/pipecd/blob/c1a06aa922a994340220a4bc5de62f6630c29b8a/pkg/app/pipedv1/cmd/piped/service/service.proto#L46-L51